conf/nginx.conf

```diff
@@ -28,5 +28,5 @@ location {LOCATION} {
 }
 
 location @wallbag2 {
-  rewrite ^ {PATH}/app.php$is_args$args;
+  rewrite ^ {PATH}/app.php?is_args$args;
```

Comment:
I'm not very familiar with nginx but if I read you well it makes sense. But it's still weird that it doesn't break anything.

You can delete the PR if you think it's irrelevant for now :-)